docs/contributing/contribute.md

```diff
@@ -0,0 +1,30 @@
+---
+uid: contrib-index
+title: Contributing
+---
+
+# Contributing to OpenBullet
+
+Thank you for your interest in contributing to the OpenBullet project! This page and its children describe the ways you can contribute, as well as some of our policies. This should help guide you through your first Issue or PR.
+
+Even if you cannot contribute code, you can still help OpenBullet! The two main things you can help with are testing and creating issues. Contributing to code, documentation, and other non-code components are all outlined in the sections below.
+
+## Reporting Issues
+
+We use GitHub extensively to track open issues, new enhancements or features, and other aspects of development.
+
+Please see the [documentation on issues](/docs/general/contributing/issues) for more information on how to submit good issues.
+
+## Developing Code
+
+The entire project consists of a C# core server, an Angular web client. If you have experience with these languages, we're always grateful for any contributions you might want to make!
```

Comment:
The entire project consists of a .NET core solution (written in C#) for the web client's API, the native Windows WPF client, and the underlying libraries. In addition, for the web client, the frontend is written in TypeScript using Angular. If you have... (same as before)